README.md

```
@@ -1,191 +1,96 @@
-### Version Updates
-```bash
-# Production
-echo 'image:\n  tag: "v1.2.3"' > charts/core-pipeline/prod.tag.yaml
-
-# Development
-echo 'image:\n  tag: "v1.2.4-dev"' > charts/core-pipeline/dev.tag.yaml
+## 📁 Structure
 
-git commit -am "deploy: update versions"
-git push origin main
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)